Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugMultiplayerMetrics.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Name/Name.h>
+#include <AzCore/Component/Entity.h>
+
+#ifdef IMGUI_ENABLED
+#   include <imgui/imgui.h>
+#   include <ImGuiBus.h>
+#   include <LYImGuiUtils/HistogramContainer.h>
+#endif
+
+namespace Multiplayer
+{
+    class MultiplayerDebugMultiplayerMetrics final
```

Comment:
Consider adding statement about responsibilities of class, esp with similarly named `MultiplayerDebugNetworkMetrics`